nucleus/src/main/resources/application.yml

```diff
@@ -4,3 +4,14 @@ spring:
       enabled: true
       path: /h2-console
   data.rest.detection-strategy: visibility
+atomist:
```

Comment:
Please remove the below webhooks. Local development will include these links appropriately.